python/tvm/relay/backend/contrib/ethosu/tir_to_cs_translator.py

```diff
@@ -36,15 +36,15 @@ class BufferType(Enum):
 
     constant = auto()
     input_or_output = auto()
-    scratch = auto()
+    runtime_allocate = auto()
```

Comment:
I'm not sure if 'runtime_allocate'/'runtime_allocation' is the best name for this as it mixes up purpose and allocation strategy. I think we can stick to 'scratch' here and rely on the 'is_runtime_allocation' field on BaseAddress to provide the further info.